cardano_node_tests/tests/test_cli.py

```diff
@@ -393,6 +394,16 @@ def _check_stake_snapshot(  # noqa: C901
                 stake_snapshot = cluster_obj.g_query.get_stake_snapshot(
                     stake_pool_ids=expected_pool_ids
                 )
+
```

Comment:
We need to check that db-sync is enabled.

Also the test `test_protocol_mode` is not marked with `dbsync`.

Reply:
I think you mean test_stake_snapshot right?